ext/web/02_event.js

```diff
@@ -1082,7 +1082,6 @@
         filename = "",
         lineno = 0,
         colno = 0,
-        error = null,
```

Comment:
`error` still exists, it just does not have a default anymore. Update this line to:

```diff
-        error = null,
+        error,
```

Reply:
Sure, updating tests currently.

Reply:
It should be fixed now.